README.md

```diff
@@ -6,11 +6,37 @@ You can obtain Circus Train from Maven Central:
+Circus Train replicates Hive tables between clusters on request. It replicates both the table's data and metadata. Unlike many other solutions it has a light touch, requiring no direct integration with Hive's core services. It can copy either entire unpartitioned tables or user defined sets of partitions on partitioned tables. A more detailed overview and the background of this project can be found in this blog post: [Replicating big datasets in the cloud](https://medium.com/hotels-com-technology/replicating-big-datasets-in-the-cloud-c0db388f6ba2). 
+
+Other features include:
+* Replication of any of the following:
+   * an entire table and all its data on each run.
+   * a subset of the data represented by a filter query (e.g. for a data set partitioned by load date you could run Circus Train once a day and replicate only the previous day's data).
+   * a subset of the data detected by looking for any changes in the data.
+   * views.
+   * table metadata (i.e. automatically replicates any DDL changes).
+* Replications between any Hadoop compatible file systems (HDFS, S3, GCS).
+* A snapshot can be taken of the source data so any changes to it while replication occurs don't result in errors or incomplete data being copied.
+* Data is replicated to a unique location at the destination so any overwriting of existing data is transparent to anyone querying the data at the same time (i.e. end users won't see inconsistent or incomplete data or receive errors). Deletion of orphaned overwritten data is handled by a configurable "house keeping" process which removes this data after an interval (to protect any "in flight" queries on the older data from errors).
+* Hive database and tables can optionally be renamed to have different names in the destination.
+* Control of how the data is copied - what file attributes are copied, bandwidth throttling etc.
+* Data consistency checking is performed as part of the replication.
+* Numerous retry mechanisms are used in order to handle the many connectivity and reliability issues that can occur when replicating across locations.
+* Transport encryption for both metadata and data.
+* Use of a Java Keystore Hadoop Credentials Provider to keep authentication keys secure.
+* Connectivity with private networks through a bastion host and direct connections.
+* Table, partition, and column statistics are also copied to preserve performance at the replica.
+* Metrics about replication (success/failure, bandwidth, bytes copied etc.) can be sent to Graphite for building dashboards, alerting etc.
+* Pluggable replication event listeners so users can implement their own functionality based on events occurring during a replication (e.g. triggering a downstream job when a replication completes). A [sample listener](https://github.com/HotelsDotCom/circus-train/tree/master/circus-train-aws-sns) that uses Amazon's Simple Notification Service (SNS) is provided as a usable reference implementation of this.
```

Comment:
A hyperlink over SNS would be great. https://aws.amazon.com/sns/

Reply:
Yeah, I didn't want this to turn into a link-fest, otherwise one could argue that above I should also hyperlink S3, Graphite, GCS etc. So for now I've just gone with linking to our listener, it's documentation should link to SNS somewhere (it currently doesn't but that's a different PR ;) )

Reply:
Ok. I will keep this in mind :P